Cargo.toml

```diff
@@ -0,0 +1,17 @@
+[package]
+name = "ttn-relay"
+version = "0.1.0"
+edition = "2021"
+publish = false
+
+[dependencies]
+anyhow = "1"
+clap = { version = "3.1.18", features = ["derive"] }
+drogue-ttn = "0.5"
+env_logger = "0.9"
+log = "0.4"
+paho-mqtt = "0.11.1"
+reqwest = { version = "0.11", features = ["blocking"] }
+serde = { version = "1", features = ["derive"] }
+serde_json = "1"
+toml = "0.5"
```

Comment:
No dependency to the firmware message encode / decode crate? Would be nice to implement it only once.

Reply:
Yeah, that part isn't implemented yet.

It's very nice though that there's the `dragino-ttn` crate for deserializing TTN payloads!

Reply:
Since the firmware decoding crate is part of the firmware repo: Should we publish it to crates.io?

Reply:
Ping @rnestler, what do you think? (An alternative would be submodules...)

Reply:
Just a git dependency should work as well, no? There is not much value in putting it on crates.io IMO.

Reply:
Ah, good point!

Reply:
Uhm, it seems the crate currently handles only encoding, but not decoding 😅

Reply:
Oh: https://github.com/gfroerli/firmware/pull/131